gstreamer-plugin/git-patch/0001-svthevcenc-Add-svthevc-encoder-element.patch

```diff
@@ -1868,10 +1872,10 @@ index 0000000..53be342
 +    plugin_init, VERSION, "GPL", GST_PACKAGE_NAME, GST_PACKAGE_ORIGIN)
 diff --git a/ext/svthevcenc/gstsvthevcenc.h b/ext/svthevcenc/gstsvthevcenc.h
 new file mode 100644
-index 0000000..be1d492
+index 000000000..ad54b9c83
 --- /dev/null
 +++ b/ext/svthevcenc/gstsvthevcenc.h
-@@ -0,0 +1,118 @@
+@@ -0,0 +1,116 @@
```

Comment:
Regen patch file can be avoid by modifying the file directly. 
Only change line 1874 and remove line 1901,1902.
Similar with ffmpeg